src/lld/dpdk/lld_dpdk_port.c

```diff
@@ -84,33 +86,48 @@ int lld_dpdk_pipeline_tap_port_add(bf_dev_port_t dev_port,
 			LOG_ERROR("ioctl SIOCSIFMTU Failed...\n");
 		close(sfd);
 
-		status = rte_swx_pipeline_port_in_config(pipe_in->p,
-							port_attrib->port_in_id,
-							"fd",
-							&params_in);
-
-		if (status) {
-			LOG_ERROR("Port in Error for Tap Port %s\n",
-				  port_attrib->port_name);
-			return BF_INVALID_ARG;
+		io_file = fopen("/tmp/iospec.io", "a+");
+		if (!io_file) {
+			LOG_ERROR("%s line:%d  Can't open file /tmp/iospec.io\n"
+				  , __func__, __LINE__);
+			return BF_INTERNAL_ERROR;
+		}
+		sprintf(buffer, "port in %d fd %d mtu %d mempool %s bsz %d\n",
```

Comment:
can we use snprintf instead of sprintf? and also initialize "buffer" to NULL above.
extend it to other places as well below.

Reply:
taken care